pkg/config/schema/schema.cue

```diff
@@ -0,0 +1,96 @@
+package config
```

Comment:
we keep calling this "config" but we're really defining the schema of a catalog/index, right?

Reply:
Correct. Perhaps, we can call this pkg `schema`.